libs/langchain/langchain/output_parsers/openai_functions.py

```diff
@@ -97,7 +150,7 @@ def validate_schema(cls, values: Dict) -> Dict:
             )
         return values
 
-    def parse_result(self, result: List[Generation]) -> Any:
+    def parse_result(self, result: List[Generation], *, partial: bool = False) -> Any:
```

Comment:
propagate partial and have underlying implementation raise if it's not supported?